### ISSUE TITLE
Will hdfs support CORS?

### ISSUE THREAD

**Issue by debuggy** [Microsoft/pai#2551]
## TL;DR
Some features in webportal need ajax requests to hdfs, which will cause a CORS problem. So will the hdfs be configured to support CORS strategies?

## Details
For example, if we want to support user upload local data to hdfs from web portal or auto fill when user inputing a hdfs path, there will be a realtime request to list the paths in hdfs.

Since webportal and hdfs are deployed in different origins, the request from webportal to hdfs will be a CORS.

From the [official doc](https://hadoop.apache.org/docs/r2.7.2/hadoop-project-dist/hadoop-common/HttpAuthentication.html#CORS), we can set hdfs config to enable the http request from webportal.
 
If we don't set CORS from hdfs, the browser will block all cors request for security, hence we have to disable all the features that needs it.

Note:
If we request through pylon, there will be no cors. However we have to support general condition without pylon.

**Comment by debuggy** [comment]
@mzmssg That's cool! When will this pr merged?
